GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -91,14 +91,16 @@ public class GVFSPlatformConstants
         {
             public static readonly char PathSeparator = Path.DirectorySeparatorChar;
 
-            public GVFSPlatformConstants(string executableExtension, string installerExtension)
+            public GVFSPlatformConstants(string executableExtension, string installerExtension, string dotGVFS)
             {
                 this.ExecutableExtension = executableExtension;
                 this.InstallerExtension = installerExtension;
+                this.DotGVFS = dotGVFS;
             }
 
             public string ExecutableExtension { get; }
             public string InstallerExtension { get; }
+            public string DotGVFS { get; }
```

Comment:
If this will not literally be a path ending in `.gvfs` for all platforms, can we give it a more meaningful name? I recommend `EnlistmentData` or something like that.